kernelci.org/content/en/docs/org/working-groups.md

```diff
@@ -19,6 +19,7 @@ as:
 ## Web dashboard
 
 **Workboard:** https://github.com/orgs/kernelci/projects/4
+**Mailing list:** [kernelci-webdashboard@groups.io](mailto:<kernelci-webdashboard@groups.io>)
```

Comment:
It's on staging:
https://static.staging.kernelci.org/docs/org/working-groups/#web-dashboard

However, it appears on the same line as **Workboard:**.  So maybe a line break, or a list or a paragraph is needed here.  I think the ideal syntax would be a definition list:
https://www.markdownguide.org/extended-syntax/#definition-lists